rosbag2_transport/test/rosbag2_transport/test_type_description_hash.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright 2023, Foxglove
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gmock/gmock.h>
+
+#include <algorithm>
+#include <future>
+#include <map>
+#include <string>
+#include <unordered_map>
+#include <unordered_set>
+#include <vector>
```

Comment:
@james-rms It seems we don't need all of this extra includes among just <string>
```suggestion
#include <string>
```